tests/linktype_name/test.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  min-version: 8
+
+args:
+- -k none --set outputs.1.eve-log.types.0.alert.packet=yes --set vars.address-groups.EXTERNAL_NET=any
+
+checks:
+  - filter:
+      count: 86
+      match:
+        event_type: alert
+        packet_info.linktype: 12
```

Comment:
this will fail on openbsd

Reply:
Why?

Reply:
this is the whole point of the ticket: this value is not cross platform

Reply:
just to be sure: this is about the numeric representation. This number 12 is not cross platform.